documentation/library_guide/kernel_templates/esimd/radix_sort.rst

```diff
@@ -215,6 +252,132 @@ Usage Examples
    1 2 3 3 3 5
    s o r t e d
 
+``radix_sort`` out of place Example
+----------------------
+
+.. code:: cpp
+
+   // possible build and run commands:
+   //    icpx -fsycl radix_sort.cpp -o radix_sort -I /path/to/oneDPL/include && ./radix_sort
+
+   #include <cstdint>
+   #include <iostream>
+   #include <sycl/sycl.hpp>
+
+   #include <oneapi/dpl/experimental/kernel_templates>
+
+   namespace kt = oneapi::dpl::experimental::kt;
+
+   int main()
+   {
+      std::size_t n = 6;
+      sycl::queue q{sycl::gpu_selector_v};
+      std::uint32_t* keys = sycl::malloc_shared<std::uint32_t>(n, q);
+      std::uint32_t* keys_out = sycl::malloc_shared<std::uint32_t>(n, q);
+
+      // initialize
+      keys[0] = 3, keys[1] = 2, keys[2] = 1, keys[3] = 5, keys[4] = 3, keys[5] = 3;
+
+      // sort
+      auto e = kt::gpu::esimd::radix_sort<false, 8>(q, keys, keys + n, keys_out, kt::kernel_param<416, 64>{}); // (4)
+      e.wait();
+
+      // print
+      for(std::size_t i = 0; i < n; ++i)
+         std::cout << keys[i] << ' ';
+      std::cout << '\n';
+      for(std::size_t i = 0; i < n; ++i)
+         std::cout << keys_out[i] << ' ';
+      std::cout << '\n';
+
+      sycl::free(keys, q);
+      sycl::free(keys_out, q);
+      return 0;
+   }
+
+**Output:**
+
+.. code:: none
+
+   3 2 1 5 3 3
+   5 3 3 3 2 1
+
+``radix_sort_by_key`` out of place Example
```

Comment:
Please change this header to: radix_sort_by_key Out of Place Example 
The header underline needs to go the full length of the header, otherwise it renders incorrectly. Please fix the length.

Reply:
fixed.